### ISSUE TITLE
Loading of castle-anim-frames is significantly slower in CGE 6.3

### ISSUE THREAD

**Issue by michaliskambi** [castle-engine/castle-engine#74]
@eugeneloza , this may interest you, since I know that you use castle-anim-frames and CGE from GitHub :)

In the commit https://github.com/castle-engine/castle-engine/commit/e240cc9ae7296dfc4f8755e5193c58d6e7e6551f , I knowingly made a change that may significantly increase the loading time (and memory consumption) of castle-anim-frames. In short, the "equality_epsilon" used by castle-anim-frames (see https://castle-engine.sourceforge.io/castle_animation_frames.php ) is temporarily ignored.

I will fix it in 2-3 days, you can watch this issue :) During that time, please 
- either use earlier engine versions, before https://github.com/castle-engine/castle-engine/commit/e240cc9ae7296dfc4f8755e5193c58d6e7e6551f ,
- or don't look too closely at the loading times of castle-anim-frames :)

Sorry for the mess, it's only temporary :)

**Comment by eugeneloza** [comment]
No problem :)
But very useful to know, as I'm trying to work with animations loading/processing at the moment. Thanks!

**Comment by michaliskambi** [comment]
I fixed the speed loss. And then I discovered that there was no speed loss... so the fix is useless. So the fix is not committed to the CGE master branch (as it makes code a bit uglier, and since it's not needed, it's better to not have it).

Sorry for the noise, looks like there was no need to notify you about this, and there was no need for me to do anything... Well, this wasn't my most productive moment :) Off to sleep now :)

More details:

After I made a fix I tested what the actual speed difference was (lost in https://github.com/castle-engine/castle-engine/commit/e240cc9ae7296dfc4f8755e5193c58d6e7e6551f and gained again in https://github.com/castle-engine/castle-engine/commit/fab51cb94854ef7a1eabf512dca7fc67676b4027 ). And there is no speed difference. Neither there is a difference in memory usage.

Of course, I should have tested this earlier, but I just assumed that https://github.com/castle-engine/castle-engine/commit/e240cc9ae7296dfc4f8755e5193c58d6e7e6551f was definitely something losing speed or increasing memory usage. It turns out it doesn't matter. The custom "epsilon" in castle-anim-frames is not actually used to gain anything anymore.

The fix is lying for now in https://github.com/castle-engine/castle-engine/compare/use-castle-anim-frames-epsilon branch , but it seems that there's no need to merge it ever to master, so eventually I'll just remove this branch.
